### PR TITLE
NOISSUE - Update SDK

### LINKED ISSUE #18: Add Groups support
Finish Group API support, what is missing.

### LINKED ISSUE #19: Create the release
After all is finished tag and create the release

### LINKED ISSUE #40: test pip install mainflux and lines from readme
Hello, @drasko!

1. What were you trying to achieve?
do: `pip install mainflux` on Windows (try and from cmd, and from vscode) python 3.10
want to use in vscode +python line `from mainflux import SDK` (like in readme)

2. What are the expected results?
to see in repo  c:\...\python310\site-packages\ folder `mainflux` with .py files
and to use line `from mainflux import SDK`

3. What are the received results?
Have three folders:
    c:\....\python310\site-packages\lib\*
    c:\....\python310\site-packages\mainflux-0.0.1.dist-info\*
    c:\....\python310\site-packages\tests\*\

and it works only if I use  `from lib.sdk import SDK` because there no dir `mainflux`

for example here is folder paho  and others (to use with import)
c:\....\python310\site-packages\paho\

4. What are the steps to reproduce the issue? 
5. In what environment did you encounter the issue?
look upper

6. Additional information you deem important:

 What I do to works  `from mainflux import SDK`
- create folder c:\....\python310\site-packages\mainflux
- copy folders lib, tests, mainflux-0.0.1.dist-info (from p.3) to folder mainflux
- create in mainflux file `__init__.py` 
- copy to file  `__init__.py`  content from file `lib\sdk.py`

so after these it works:
```
from mainflux import SDK
default_url = "http://localhost"      <- what is this line for? If it have to change default_url - it's not working
sdk = SDK()

# Example to create an account
mf_resp = sdk.users.create({"email": "<user_email>", "password": "<user_password>"})
if mf_resp.error.status == 0:
    print(mf_resp.value)
else:
    print(mf_resp.error.message)
```

but I use user.login  and change default_url directly in `__init__.py`
image

### PR DESCRIPTION
**Pull request by rodneyosodo**
### What does this do?
It updates sdk to include groups, certs and boostrap

### Which issue(s) does this PR fix/relate to?
Resolves #18 
Resolves #19 
Resolves #40 

### List any changes that modify/break current functionality
changes from
```py
from lib import sdk
```
to
```py
from mainflux import sdk
```
### Have you included tests for your changes?
Yes

### Did you document any new/modified functionality?
Yes

### Notes
N/A

### CODE REVIEW

**Comment by drasko** [comment]
@Musilah and @fbugarski please review.

Review only visually, do not test - it will not work anyways.

We will merge and break, then @Musilah will fix.

File: mainflux/sdk.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+from mainflux import messages
+from mainflux import channels
+from mainflux import groups
+from mainflux import keys
+from mainflux import boostrap
+from mainflux import certs
+
+import requests
+
+default_url = "http://localhost"
```
Comment (fbugarski) [inline mainflux/sdk.py:12]: Change constant to uppercase letters: DEFAULT_URL

File: mainflux/sdk.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+
+class SDK:
+    def __init__(
+        self,
+        users_url=default_url,
```
Comment (fbugarski) [inline mainflux/sdk.py:18]: same here

File: mainflux/boostrap.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
+import requests
+
+from mainflux import response
+from mainflux import errors
+from mainflux import utils
+
+
+class Bootstrap:
+    configs_endpoint = "configs"
```
Comment (fbugarski) [inline mainflux/boostrap.py:9]: Uppercase letters for constants, i think that is practise in Python, 
@drasko what do you say ?

**Review by fbugarski — CHANGES_REQUESTED** [review]

File: mainflux/boostrap.py (hunk 3 above)
Comment (drasko) [reply mainflux/boostrap.py:9]: @fbugarski correct, this is idiomatic way of writing constants in Python, and is documented in their code style PEP: https://peps.python.org/pep-0008/#constants

**Comment by drasko** [comment]
I will merge this one, although it will break the functionality. But @Musilah will go right after and correct everything with her PR.

**Review by Musilah — APPROVED** [review]

**Pull request merged**